CMakeLists.txt

```diff
@@ -242,27 +242,22 @@ install(
   FILES ${CMAKE_CURRENT_BINARY_DIR}/${EXT_CONTROL_FILE}
   DESTINATION "${PG_SHAREDIR}/extension")
 
-find_program(PGINDENT pgindent
-  HINTS ${PG_SOURCE_DIR}
-  PATH_SUFFIXES src/tools/pgindent
-  DOC "Format C code according to PostgreSQL standards")
-
-# Configuration for running pgindent
-if (PGINDENT)
-  message(STATUS "Using pgindent ${PGINDENT}")
-  file(WRITE ${CMAKE_BINARY_DIR}/pgindent_excludes "^\\..*/\n")
-
-  add_custom_command(OUTPUT typedefs.list
-    DEPENDS ${PROJECT_NAME}
-    COMMAND sh ${PROJECT_BINARY_DIR}/scripts/generate_typedefs.sh > ${PROJECT_BINARY_DIR}/typedefs.list)
-  add_custom_target(pgindent
-    COMMAND ${PGINDENT} -typedefs typedefs.list -excludes=${PROJECT_BINARY_DIR}/pgindent_excludes -code-base=${PROJECT_SOURCE_DIR}/src
-    COMMAND ${PGINDENT} -typedefs typedefs.list -excludes=${PROJECT_BINARY_DIR}/pgindent_excludes -code-base=${PROJECT_SOURCE_DIR}/test/src
-    COMMAND ${PGINDENT} -typedefs typedefs.list -excludes=${PROJECT_BINARY_DIR}/pgindent_excludes -code-base=${PROJECT_SOURCE_DIR}/tsl/src
-    DEPENDS ${PROJECT_BINARY_DIR}/typedefs.list)
-else ()
-  message(STATUS "Install pgindent to be able to format C code: https://github.com/postgres/postgres/tree/master/src/tools/pgindent")
-endif (PGINDENT)
+find_program(CLANG_FORMAT clang-format
+  DOC
+  "The path to clang-format")
+
+if (NOT CLANG_FORMAT)
+  message(STATUS "Using docker based clang-format")
+   add_custom_target(format
```

Comment:
I think you should do `find_program` on docker here as well if docker is a dependency in CMake.